tests/e2e/cht-form/default/pregnancy-danger-sign-follow-up.wdio-spec.js

```diff
@@ -15,8 +15,19 @@ describe('cht-form web component - Pregnancy Danger Sign Form', () => {
     const title  = await genericForm.getFormTitle();
     expect(title).to.equal('Pregnancy danger sign follow-up');
 
-    await pregDangerSignFollowUpForm.selectVisitedHealthFacility(true);
-    await pregDangerSignFollowUpForm.selectDangerSigns(true);
+    await commonEnketoPage.selectRadioButton('Did the woman visit the health facility as recommended?', 'Yes');
+    await commonEnketoPage.selectRadioButton('Is she still experiencing any danger signs?', 'Yes');
+    await commonEnketoPage.selectRadioButton('Vaginal bleeding', 'Yes');
+    await commonEnketoPage.selectRadioButton('Fits', 'Yes');
+    await commonEnketoPage.selectRadioButton('Severe abdominal pain', 'Yes');
+    await commonEnketoPage.selectRadioButton('Severe headache', 'Yes');
+    await commonEnketoPage.selectRadioButton('Very pale', 'Yes');
+    await commonEnketoPage.selectRadioButton('Fever', 'Yes');
+    await commonEnketoPage.selectRadioButton('Reduced or no fetal movements', 'Yes');
+    await commonEnketoPage.selectRadioButton('Breaking of water', 'Yes');
+    await commonEnketoPage.selectRadioButton('Getting tired easily', 'Yes');
+    await commonEnketoPage.selectRadioButton('Swelling of face and hands', 'Yes');
+    await commonEnketoPage.selectRadioButton('Breathlessness', 'Yes');
```

Comment:
Overall I think this is an improvement so don't take this as a blocker... but I think there's still value in bundling these up in the page object if it reduces some of the verbiage in the test. As in, this test would call the predanger sign follow up form page, and the predanger sign follow up form page would call the common enketo page.

What do you think?

Reply:
It will definitely look better, and those danger signs are set in 2 or test, I will create the file and isolate them in a separate function. Thank you!